doc/transfer-format.rst

```diff
@@ -28,82 +28,83 @@ still easily be processed by OpenPGP tools. In particular, GnuPG ignores the
 header before the armored data starts, so a regular call to :code:`gpg
 --decrypt` still works.
 
-An exemplary secret key backup MIME part would look as such:
+An exemplary Autocrypt Setup Message's MIME part would look like the
+text below.  This Autocrypt Setup Message can be unlocked with the
+Setup Code ``D9VN-RD7A-7T3B-BWG7-LEPY-3NYT``.
 
 .. code-block:: html
 
   Content-Type: application/pgp-key-backup; name="Autocrypt-Key for name-at-example.org.html"
   Content-Description: OpenPGP Digital Signature
   Content-Disposition: attachment; filename="Autocrypt-Key for name-at-example.org.html"
 
-  <html><body><h1>Autocrypt Key</h1><p>
+  <html><body><h1>Autocrypt Key for <tt>example.org</tt></h1><p>
 
   This is an Autocrypt Key file. When imported into an Autocrypt capable E-Mail
   client with its backup code, it can be used to read encrypted E-Mail sent to
-  name@example.org.
+  <tt>name@example.org</tt>.
 
   </p><pre>
   -----BEGIN PGP MESSAGE-----
+  Autocrypt-Setup-Message: v1
```

Comment:
If this was in a mail header it would allow identifying the email and rely on existing header parsing rather than requiring special treatment.